nodewatcher/modules/frontend/ip_space/static/js/main.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(function(){
+    var svgContainer = d3.select("#ipspace");
+    var size = svgContainer.node().getBoundingClientRect().width;
+    svgContainer.style("height", size + 'px');
+    var main_subnet = "10.0.0.0/8";
```

Comment:
Why is this hardcoded? Maybe fetch this from template? Like you are doing for `#cidr`? So initialize from it.